src/api/app/models/comment.rb

```diff
@@ -51,6 +51,16 @@ def unused_parent?
     parent && parent.user.is_nobody? && parent.children.empty?
   end
 
+  def moderated?
+    moderated_at || moderated_by
+  end
+
+  def moderate(state)
+    moderated_at = state ? Time.now : nil
+    moderated_by = state ? User.session : nil
+    save!
+  end
```

Comment:
https://github.com/pienkowb/active_record-events

Reply:
@hennevogel we wonder if it's worth introducing the gem to handle `moderated_at` as we still have to handle `moderator_id` manually anyway...

Reply:
I just wanted to point it out, you decide.